DARTS/perform_pred.py

```diff
@@ -1,57 +1,48 @@
+#!/usr/bin/env python
+
 import torch
 import torch.nn as nn
 import numpy as np
 import os
+from os.path import abspath, join as pjoin
 import nibabel
-from models.dense_unet_model import Single_level_densenet, Down_sample, Upsample_n_Concat, Dense_Unet
-# from models.dense_unet_model import *
-from utils import load_data, create_one_hot_seg, back_to_original_4_pred, back_to_original_4_prob, pickling
-from models.unet import Unet,Downsample_block,Upsample_block
+from DARTS.models.dense_unet_model import Single_level_densenet, Down_sample, Upsample_n_Concat, Dense_Unet
+from DARTS.utils import load_data, create_one_hot_seg, back_to_original_4_pred, back_to_original_4_prob, pickling
```

Comment:
After `pip install DARTSeg`, `dense_unet_model.py` already appears in `~/miniconda3/lib/python3.*/site-packages/DARTS/`. Given that, importing from local `models` directory is improper and unequivocal. Again, following standard practice, I reorganized the modules and defined imports accordingly.